test/integration/smart_proxy_test.rb

```diff
@@ -1,6 +1,10 @@
 require 'test_helper'
 
 class SmartProxyIntegrationTest < ActionDispatch::IntegrationTest
+  setup do
```

Comment:
This should be in a teardown for any tests that are adding pagelets (preferably in a context block).